backend/tracim_backend/lib/utils/utils.py

```diff
@@ -392,6 +393,31 @@ def email_link(self) -> str:
         return "mailto:{email_address}".format(email_address=self.email_address)
 
 
+def load_apps() -> typing.List[TracimApplication]:
+    import tracim_backend.applications as apps_modules
+
+    tracim_apps = []
+    for app_config_path in find_direct_submodule_path(apps_modules):
+        module = importlib.import_module("{}.application".format(app_config_path))
+        tracim_apps.append(module.create_app())
+    return tracim_apps
+
+
+def find_direct_submodule_path(module: types.ModuleType) -> typing.List[str]:
+    """
+    Get path of submodules of given module
+    :param module: module to check
+    :return: list of path of direct submodule
```

Comment:
please add examples. is this something like `applications.agenda`? `applications/agenda`? is a value like `applications/agenda/somesubmodule` possible?

Reply:
documentation should be clear enough in order to understand how to use the function without to have to read its implementation

Reply:
done.